auth-api/src/auth_api/utils/enums.py

```diff
@@ -353,3 +354,21 @@ class NotificationTypes(Enum):
     DETAILED_CONFIRMATION_PRODUCT = 'productConfirmationNotification'
     DETAILED_APPROVED_PRODUCT = 'productApprovedNotificationDetailed'
     DETAILED_REJECTED_PRODUCT = 'productRejectedNotificationDetailed'
+
+
+class NRActionCodes(Enum):
+    """Name Request Action Codes."""
+
+    AMALGAMATE = 'AML'
+    ASSUMED = 'ASSUMED'  # FUTURE: should be AS (as in LEAR)?
+    CHANGE_NAME = 'CHG'
+    CONVERSION = 'CNV'  # aka Alteration
+    DBA = 'DBA'  # doing business as
+    MOVE = 'MVE'  # continuation in
+    NEW_BUSINESS = 'NEW'  # incorporate or register
+    RESTORE = 'REH'  # restore or reinstate
+    RENEW = 'REN'  # restore with new name request
+    RESTORATION = 'REST'  # FUTURE: unused? delete?
+    RESUBMIT = 'RESUBMIT'  # FUTURE: unused? delete?
+
+    INFO = 'INFO'  # special value for sub-menu
```

Comment:
You don't need INFO in the API; it's only for Namerequest UI.

Reply:
Updated